index.html

```diff
@@ -133,6 +133,190 @@ <h2 id="headline-text">I'm Kanza. Glad to see you!</h2>
             ></a>
```

Comment:
Kindly, I suggest that the icon listings use UL/LI lists, your site will be better structured